modules/text/include/opencv2/text/ocr.hpp

```diff
@@ -538,6 +656,403 @@ CV_EXPORTS_W Ptr<OCRBeamSearchDecoder::ClassifierCallback> loadOCRBeamSearchClas
 
 //! @}
 
-}
-}
+
+//Classifiers should provide diferent backends
+//For the moment only caffe is implemeted
```

Comment:
Isn't this comment inaccurate? Now you have both caffe and DNN implemented.

Reply:
Updated in new code